src/requestBuilder.js

```diff
@@ -72,7 +72,7 @@ var requestBuilder = {
       additionalParams.numericFilters = numericFilters;
     }
 
-    return merge(state.getQueryParams(), additionalParams);
+    return merge({}, state.getQueryParams(), additionalParams);
```

Comment:
What happens without prepending {} ?

Reply:
This was when I initially wasn't sure whether to modify the target object. None of the usages rely on this though, so I made that clear by creating a fresh object. (like you'd do in Object.assign usually)